### DISCUSSION TITLE
Could remotestorage be integrated?

### DISCUSSION THREAD

**Discussion by Narvey** [learn-awesome/learndb#67] [Ideas]
Just wondering if https://remotestorage.io could be feasible (at least in some point in the future) as a sort of cloud-backup of the localstorage data this app generates? I might be able to help a little if the project believes remotestorage is a good idea.

**Comment by nileshtrivedi** [comment]
@Narvey This is the first time I've come across remotestorage. The idea is nice. Privacy can also be ensured by storing only encrypted data. There's a minor concern of the server later going rogue and keeping users' data hostage. Ability to self-host does help allay that concern a bit. 

So, in principle, I'm for it. 👍  But I'm not sure if this is high priority at this point. I'd like to see more users demand this feature before building it.
